navapbc/wic-mt-demo-project-eligibility-screener
# Minor follow up changes for #142, #144, #146

## Ticket

N/A

## Changes
> What was added, updated, or removed in this PR.

- Use more [narrow typing for Button and ButtonLink components to reduce manual storybook changes](https://github.com/navapbc/wic-mt-demo-project-eligibility-screener/pull/144#discussion_r1009636387)
- Update comments (discussion references [1](https://github.com/navapbc/wic-mt-demo-project-eligibility-screener/pull/146#discussion_r1009671754), [2](https://github.com/navapbc/wic-mt-demo-project-eligibility-screener/pull/142#discussion_r1007690941), [3](https://github.com/navapbc/wic-mt-demo-project-eligibility-screener/pull/146#discussion_r1009674249))
- [Rename variable naming for clarity](https://github.com/navapbc/wic-mt-demo-project-eligibility-screener/pull/142#discussion_r1008511847)
- [Drop extraneous else](https://github.com/navapbc/wic-mt-demo-project-eligibility-screener/pull/142#discussion_r1009601544)

## Context for reviewers
> Testing instructions, background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers. Explain how the changes were verified.

This PR incorporates some minor feedback from the following PRs:
- #142 
- #144 
- #146 

## Testing
> Screenshots, GIF demos, code examples or output to help show the changes working as expected. ProTip: you can drag and drop or paste images into this textbox.

First test: These are non-functional changes, so running `yarn test` should prevent any regressions.

Second test: Check the storybook controls work as expected for Button and ButtonLink (there should be dropdowns with all acceptable options listed):
  1. Checkout this branch
  2. Build and run the container: `docker-compose up -d --build`
  4. Build and run storybook: `docker-compose exec nextjs yarn storybook`
  5. Navigate to localhost:6006
  6. Confirm that the landing page is the MWDP index page and not the Accordion component
  7. Remove the container when you're done: `docker-compose down`
